core/Decode.cpp

```diff
@@ -1,7 +1,7 @@
 // <Decode.cpp> -*- C++ -*-
 
 #include "Decode.hpp"
-#include "fusion/FusionTypes.hpp"
+#include "fsl_api/FusionTypes.h"
```

Comment:
Curious, why the change in name?

Reply:
In the early days fusion was accurate, I have cases where I want to apply the language to fracture, some RV extension and for vector. 

I chose fsl_api and fsl_interp to separate the namespaces. 

It could have been fsl and fsl_interp for the symmetry and to quiet my OCD.

Olympia is the lead use case, so if you prefer I can change fsl_api to fsl.